src/presentation/pages/SignUp/index.tsx

```diff
@@ -32,6 +32,25 @@ const SignUp: React.FC<SignUpProps> = ({
     mainError: '',
   })
 
+  const [strength, setStrength] = useState('');
+  const evaluatePasswordStrength = (value) => {
+
+    //Conditions for password strength
+    const strongRegex = new RegExp("^(?=.{8,})(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9])(?=.*\\W).*$");
+    const mediumRegex = new RegExp("^(?=.{6,}).*$");
+
+    //logic
+    if(value.length<4){
```

Comment:
try to improve this logic due to the object calisthenics, and add strings to an enum